macros/get_enabled_unioned_models.sql

```diff
@@ -0,0 +1,33 @@
+{% macro get_enabled_unioned_models() %}
+
+{% set unioned_models = [
+    'sales_receipt', 
+    'credit_memo',
+    'bill',
+    'deposit',
+    'invoice',
+    'transfer',
+    'journal_entry',
+    'payment',
+    'refund_receipt',
+    'vendor_credit'] %}
```

Comment:
I apologize for this request, can we alphabetize these

Reply:
Fixed now!